src/einsteinpy/symbolic/predefined/__init__.py

```diff
@@ -8,5 +8,5 @@
 from .find import find
 from .godel import Godel
 from .janis_newman_winicour import JanisNewmanWinicour
-from .minkowski import Minkowski
+from .minkowski import Minkowski, MinkowskiPolar
```

Comment:
```suggestion
from .minkowski import Minkowski, MinkowskiCartesian, MinkowskiPolar
```

In tune with the prior suggestions